### DISCUSSION TITLE
GitHub keeps uploading the old verison

### DISCUSSION THREAD

**Discussion by craigcooper11** [community/community#21884] [New to GitHub]
<p>I am trying to add an angularjs project to gethub, I follow the sets and it keeps putting the old version.</p>

**Comment by ThirdEyeRose** [comment]
<p>Hi <a class="mention" href="/utsa123432">@utsa123432</a>,</p>
<p>When you say that it keeps uploading the old version, do you mean that you have two versions on your local machine and it keeps uploading the older version when you try to push the project to GitHub? Could you tell us what steps you are taking from beginning to end?</p>
<p>Thanks!</p>

**Comment by craigcooper11** [comment, marked as answer]
<p>Hi! Thanks for responding. I figured it out, I wasn’t committing my files after adding them! All is good :slight_smile:</p>

**Comment by ThirdEyeRose** [comment]
<p>Great! Glad to hear that you figured it out.</p>
